### ISSUE TITLE
Identical Fingerprints with 2G and 5G possible?

### ISSUE THREAD

**Issue by CableMate** [nzymedefense/nzyme#1010]
Is it possible that there are 2 identical fingerprints for 2G and 5G?
The low rates with the 5G MAC are also strange.
The WLAN-Hardware are two AWUS036ACM. One for 2G one for 5G channels
![Network-Details-2G](https://github.com/nzymedefense/nzyme/assets/68850559/7a692c2b-2001-4498-9896-1f234492cb98)
![Network-Details-5G](https://github.com/nzymedefense/nzyme/assets/68850559/b72ccf53-2f53-430e-b5ad-4047ba485df3)

**Comment by lennartkoopmann** [comment]
Yes, that is possible. The fingerprints are calculated from attributes that are not necessarily linked to the band in operation.

Can you try to compare the reported rates to what Wireshark or another tool is reporting?

**Comment by CableMate** [comment]
I can try, as soon I see another occurence.
